examples/log-ingestion/docker-compose-dataprepper.yaml

```
@@ -0,0 +1,15 @@
+version: ‘3.7’
```

These are backticks. Does this work with YAML? Can you make these single quotes?

```suggestion
version: '3.7'
```